document indexed_only (#284)
qdrant/landing_page: qdrant-landing/content/documentation/concepts/search.md

* `hnsw_ef` - value that specifies `ef` parameter of the HNSW algorithm.
* `exact` - option to not use the approximate search (ANN). If set to true, the search may run for a long as it performs a full scan to retrieve exact results.

> * `indexed_only` - this option allows to disable search in those segments where vector index is not built yet. This may be useful if you want to minimize the impact to the search performance during the parallel update of the collection. Using this option may lead to a partial result if the collection is not fully indexed yet, consider using it only if eventual consistency is acceptable for your use case.

Reviewer: Nitpick, but this may be a _bit_ easier to understand:

```suggestion
* `indexed_only` - this option allows to disable search in those segments where vector index is not built yet. This may be useful if you want to minimize the impact to the search performance whilst the collection is also being updated. Using this option may lead to a partial result if the collection is not fully indexed yet, consider using it only if eventual consistency is acceptable for your use case.
```
